packages/react-composites/playwright.config.common.ts

```diff
@@ -52,6 +52,10 @@ const LOCAL_REPORTERS: ReporterDescription[] = [['list']];
 const SECOND = 1000;
 const MINUTE = 60 * SECOND;
 
+// Using chromium useragent with a very high version to avoid breaking the unsupportedBrowser page
+const DESKTOP_USER_AGENT = 'Windows Chrome/999.0.0.0';
```

Comment:
Do we have a single hermetic test for this updated warning page? like set the expired browser version then test if the page shows up

Reply:
Yep we do! https://github.com/Azure/communication-ui-library/blob/5cae388f3bd54c64cbc999696f42fc474fa0dc31/packages/react-composites/tests/browser/call/hermetic/UnsupportedBrowserPage.test.ts